pkg/nb/types.go

```diff
@@ -718,3 +720,7 @@ func UInt64ToBigInt(value uint64) BigInt {
 		N:    int64(value % petaInBytes),
 	}
 }
+
+func (params CreateSystemParams) String() string {
```

Comment:
as discussed, to make it more reusabel let's implement a Stringer for a more basic string type